optout/s3_file_handler.go

```diff
@@ -178,7 +179,7 @@ func (handler *S3FileHandler) createSession() (*session.Session, error) {
 		config.Endpoint = &handler.Endpoint
 	}
 
-	if handler.AssumeRoleArn != "" {
+	if handler.AssumeRoleArn != "" && os.Getenv("LOCAL_STACK_ENDPOINT") == "" {
```

Comment:
Needed this hack to get unit tests to work correctly. Localstack was returning "Expired credentials" errors when attempting to assume the fake BFD role, but those parameters are required to exist in order to run the import code.

Based on documentation, this doesn't seem to be the expected behavior (we should be able run assume-role commands in Localstack without any permission errors) but this was a quick workaround to make tests work.